script/server

```diff
@@ -6,5 +6,37 @@ set -e
 # check that we are in the expected directory
 cd `dirname $0`/..
 
-./manage.py migrate
-./manage.py runserver 0.0.0.0:8000
+ACTION=$1
+[ -n "$ACTION" ] || ACTION="--local"
+
+case "$ACTION" in
+    --local)
+        script/bootstrap
+        source .env
+        if [ "$DEVENV" == "docker" ] ; then
+            echo "==> Removing any stopped or crashed containers..."
+            docker-compose rm
```

Comment:
The database files should be persisted in the `pgdata` volume, so that shouldn't be necessary. I'd argue that it should be possible to remove the containers as a matter of course - no state should be persisted solely within a container as these should be ephemeral.

Reply:
Oh, right, yes of course, if the data's kept that's fine, dur.